LL 6326 - LLM add twitter banner

klambert-ledger opened the pull request:
Adding a Twitter banner inside the Platform Catalog.
Banner is closable and hashtag pressable to go to a pre filled twitter form (through the browser or the Twitter App).

### Screenshots
<details>
  <summary>iOS 📸 </summary>

![Simulator Screen Shot - iPhone 11 - 2021-07-07 at 16 53 10](https://user-images.githubusercontent.com/86959861/124966070-1d00c900-e023-11eb-9c01-080a4005ef6a.png)
</details>

### Type

Feature

### Context

[LL-6326](https://ledgerhq.atlassian.net/browse/LL-6326)

### Parts of the app affected / Test plan

Platform/Catalog
Translations

IAmMorrow reviewed (dismissed):
Good first contribution ! 🥳 
Welcome to the team !

juan-cortes commented on src/screens/Platform/CatalogBanner.js:
```diff
@@ -34,15 +36,25 @@ const styles = StyleSheet.create({
   title: {
     fontSize: 18,
-    marginBottom: 16,
+    marginBottom: 7,
```
We _try_ to follow 2,4,8,12,16,32 for distance so this should be an 8 even if on figma it's a 7, but 🤷🏼‍♂️

klambert-ledger replied:
Yes that was the value on figma. I'll fix this right now ! 👍

juan-cortes reviewed (dismissed):
lgtm

ychen-ledger reviewed (dismissed):
Validated on ios side: 

- Iphone 6s 
- simulator iphone 11
- Nano X

klambert-ledger commented:
Just moved the url to config to fit more with the review made on LLD (https://github.com/LedgerHQ/ledger-live-desktop/pull/3969) with the same feature. 

Sry @ychen-ledger if you have to retest it again 😞 (you really shouldn't have to, I just moved a variable and tested it before pushing it).

ychen-ledger reviewed (commented):
@klambert-ledger No worries, feel free to ping me if any further test needed😄.

achabrolin-ledger reviewed (approved):
lgtm
